### DISCUSSION TITLE
插件命令没有回复

### DISCUSSION THREAD

**Discussion by Dr-WeiAL** [nonebot/discussions#270] [问答] [answered]
安装了HarukaBot和Apex API Query
私聊机器人 帮助和/maprotation都没有反应
HarukaBot的主人QQ号和Apex API Query的API 密钥都填入了.env.prod中
后台提示这些
![image](https://user-images.githubusercontent.com/47446553/207209739-1ec4ed86-0ecd-402d-8a8d-089db30e3b86.png)

**Comment by yanyongyu** [comment, marked as answer]
你当前启动的环境是prod吗

**Reply by Dr-WeiAL** [reply]
好吧之前是dev，我翻到了 https://github.com/SK-415/HarukaBot/issues/89 这里改成prod后就能用了

**Reply by Dr-WeiAL** [reply]
又出现了新问题(
HarukaBot可以用了
但Apex API Query的命令没有反应，后台也没提示任何东西
![image](https://user-images.githubusercontent.com/47446553/207212750-e7431c9d-bea7-4dd9-a5a0-40b15eaf8263.png)
![image](https://user-images.githubusercontent.com/47446553/207212778-d3ae80a5-dcc8-496d-98c0-58faed526650.png)
![image](https://user-images.githubusercontent.com/47446553/207212837-30db6fcc-b6ac-4c4f-80ca-56107fdf2a7f.png)

**Reply by Dr-WeiAL** [reply]
已经在群里解决了，这个插件自己的问题
